src/EnergyPlus/EconomicTariff.cc

```diff
@@ -3868,11 +3904,24 @@ namespace EconomicTariff {
 			PreDefTableEntry( pdchLeedEcsTotal, "Electricity", elecTotalCost, 2 );
 			PreDefTableEntry( pdchLeedEcsTotal, "Natural Gas", gasTotalCost, 2 );
 			PreDefTableEntry( pdchLeedEcsTotal, "Other", otherTotalCost, 2 );
+			// show district energy if used
+			if ( distCoolTotalEne != 0 ) {
+				PreDefTableEntry( pdchLeedEtsVirt, "District Cooling", distCoolTotalCost / distCoolTotalEne, 3 );
+				PreDefTableEntry( pdchLeedEtsEneUnt, "District Cooling", convEneStrings( distCoolUnits ) );
+				PreDefTableEntry( pdchLeedEtsDemUnt, "District Cooling", convDemStrings( distCoolUnits ) + demWindowStrings( distCoolDemWindowUnits ) );
+				PreDefTableEntry( pdchLeedEcsTotal, "District Cooling", distCoolTotalCost, 2 );
+			}
+			if ( distHeatTotalEne != 0 ) {
+				PreDefTableEntry( pdchLeedEtsVirt, "District Heating", distHeatTotalCost / distHeatTotalEne, 3 );
+				PreDefTableEntry( pdchLeedEtsEneUnt, "District Heating", convEneStrings( distHeatUnits ) );
+				PreDefTableEntry( pdchLeedEtsDemUnt, "District Heating", convDemStrings( distHeatUnits ) + demWindowStrings( distHeatDemWindowUnits ) );
+				PreDefTableEntry( pdchLeedEcsTotal, "District Heating", distHeatTotalCost, 2 );
+			}
 			// save the total costs for later to compute process fraction
 			LEEDelecCostTotal = elecTotalCost;
 			LEEDgasCostTotal = gasTotalCost;
-			LEEDothrCostTotal = otherTotalCost;
-			PreDefTableEntry( pdchLeedEcsTotal, "Total", elecTotalCost + gasTotalCost + otherTotalCost, 2 );
+			LEEDothrCostTotal = distCoolTotalCost + distHeatTotalCost + otherTotalCost;
+			PreDefTableEntry( pdchLeedEcsTotal, "Total", elecTotalCost + gasTotalCost + distCoolTotalCost + distHeatTotalCost + otherTotalCost, 2 );
```

Comment:
So with this change, does it correct previous outputs?  It kinda looks like for district heating/cooling cases, previously the "Total" would have been incorrectly not including those rates, and now they will be correct?